ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -111,6 +115,30 @@ images:
     RUN go mod vendor
   from: assisted-service-centos9
   to: assisted-service-subsystem
+- dockerfile_literal: |
+    FROM base
+    ENV VIRTUAL_ENV=/opt/venv
+    ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+    COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
+    RUN cd / && /assisted-service/hack/setup_env.sh spectral && \
+        /assisted-service/hack/setup_env.sh jq && \
+        /assisted-service/hack/setup_env.sh kustomize && \
+        go install github.com/golang/mock/mockgen@v1.5.0 && \
+        go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2 && \
+        go install golang.org/x/tools/cmd/goimports@v0.1.5 && \
+        chmod 775 -R $GOPATH
+    RUN curl --retry 5 -L https://github.com/operator-framework/operator-sdk/releases/download/v1.10.1/operator-sdk_linux_amd64 --output /tmp/operator-sdk_linux_amd64 && \
+        chmod +x /tmp/operator-sdk_linux_amd64 && \
+        install /tmp/operator-sdk_linux_amd64 /usr/local/bin/operator-sdk
+    RUN python3 -m venv ${VIRTUAL_ENV:-/opt/venv} && \
```

Comment:
it seems redundant to have a default value when we clearly defined it above (with ``ENV VIRTUAL_ENV=/opt/venv``), no?